bambi/plots/plot_cap.py

```diff
@@ -244,7 +255,7 @@ def plot_cap(
     for ax in axes.ravel():  # pylint: disable = redefined-argument-from-local
         ax.set(xlabel=main, ylabel=ylabel)
 
-    return fig, axes
+    return (fig, axes), idata, cap_data
```

Comment:
I'm not so sure about returning `idata` and `cap_data` as well. I see it would be nice because it allows you to do more things with that, but i think it could be surprising for users who are not familiar with these data structures. So perhaps we can have an optional `return_data` argument?

Reply:
Ahhh, yes I forgot to delete this code. I was returning the data objects to inspect myself. Thanks for that!